docs/Data_correction_documentary_cleared.ipynb

```diff
@@ -0,0 +1,299 @@
+    "\n",
+    "$$\n",
+    "O^{cell}(\\lambda)=O^{0, cell}\\cdot\\lambda\n",
+    "$$.\n",
+    "\n",
+    "For an in-situ quick data correction, a measure of $p$ during filling the cell can serve as input to above equation, leading to \"workflow 1\". However, a measurement of the opacity by neutron transmission in a depolarized cell condition yields more precise values and can be performed at the end of the cell-lifetime during the user's neutron experiments, i.e., leading to \"workflow 2\". For both workflows, the constant prefactor $O^{0, cell}$ will be determined either from workflow 1 or 2.\n",
+    "\n",
+    "- Note: $l$ is assumed constant over the area of the $^3$He-cell. In case of too large cell volumes, only the fraction with constant $l$/ constant $O$ will be used for the neutron measurements. The same assumption holds for both SANS-cells as well as wide-angle cells\n",
+    "- Note: until now, we have treated $T^{g, cell}$ as constant parameter and ignored its wavelength-dependence, as its variation with wavelength is small enough to not impact the correction procedure. However, to keep it generic and enable to insert a wavelength-dependent $T^{g, cell}(\\lambda)$, the dependence on wavelength will be pre-characterized (during fabrication of the cell) and inserted as function into this workflow instead of a scalar.\n",
+    "\n",
+    "$\\mathbf{Workflow 2}$:<br>\n",
+    "A more precise data correction is based on a measure of $p$ by a direct-beam measurement through the depolarized cell (i.e., $P_{^3He}(t) \\rightarrow 0$)(Ref. 2), leading to \"workflow 2\". For $P_{^3He}(t) \\rightarrow 0$ and an unpolarized incident beam on the depolarized cell(i.e., $\\hat{T}^{depol} = (\\hat{T}^++\\hat{T}^-)/2$), the neutron transmission becomes:\n",
+    "\n",
+    "$$\n",
+    "T^{depol, cell}(\\lambda)=T^{g, cell}(\\lambda)e^{-O^{0, cell}\\cdot\\lambda}\n",
+    "$$\n",
+    "\n",
+    "This leads to following derivation by neutron experiment:\n",
+    "\n",
+    "$$\n",
+    "\\frac{I^{DB,depol,cell}_{\\lambda}}{I^{DB,no-cell}_{\\lambda}}=T^{g, cell}(\\lambda)e^{-O^{0, cell}\\cdot\\lambda}\n",
+    "$$\n",
+    "\n",
+    "For a measurement of $I^{DB,depol,cell}_{\\lambda}$, the $^3$He cell needs to be depolarized at the end of its lifetime and a direct-beam measurement through each cell (and the other cell out of beam) has to be performed without sample. \n",
+    "\n",
+    "- Note: Workflow 2 should overwrite workflow 1 opacity after the experiment\n",
+    "- Note: Different to the sample data, $I^{DB,depol,cell}$ and $I^{DB,no-cell}$ are not time-dependent\n",
+    "- Note: It is important to fit $I_{depol}/I_0$ instead of $I_{depol}$, such that all the parameters in the fit-functions are scalars.\n"
```

Comment:
What is I_0? I can't find a definition.

Reply:
I forgot to change the name of this in the note, sorry. I_0 is now I^{DB, no-cell}_{\lambda}, I will change it